### ISSUE TITLE
Integration Test Fails : TestMonitorCreateAndDelete IncludeTags

### ISSUE THREAD

**Issue by dharada1** [zorkian/go-datadog-api#209]
ran on my local, with my datadog api & app key

also master https://travis-ci.org/zorkian/go-datadog-api/builds/490935059 fails

```
=== RUN   TestMonitorCreateAndDelete
--- FAIL: TestMonitorCreateAndDelete (1.26s)
	monitors_test.go:21: 
			Error Trace:	monitors_test.go:21
			Error:      	Not equal: 
			            	expected: &datadog.Monitor{Creator:(*datadog.Creator)(0xc420246720), Id:(*int)(0xc42024a828), Type:(*string)(0xc420218450), Query:(*string)(0xc420218430), Name:(*string)(0xc420218440), Message:(*string)(0xc420218420), OverallState:(*string)(0xc420218460), OverallStateModified:(*string)(nil), Tags:[]string{}, Options:(*datadog.Options)(0xc420160180), State:datadog.State{Groups:map[string]datadog.GroupData(nil)}}
			            	actual  : &datadog.Monitor{Creator:(*datadog.Creator)(0xc420246680), Id:(*int)(0xc42024a718), Type:(*string)(0xc4202185f0), Query:(*string)(0xc420218580), Name:(*string)(0xc4202185a0), Message:(*string)(0xc420218590), OverallState:(*string)(0xc4202185e0), OverallStateModified:(*string)(nil), Tags:[]string{}, Options:(*datadog.Options)(0xc420160400), State:datadog.State{Groups:map[string]datadog.GroupData(nil)}}
			            	
			            	Diff:
			            	--- Expected
			            	+++ Actual
			            	@@ -29,3 +29,3 @@
			            	   ThresholdWindows: (*datadog.ThresholdWindows)(<nil>),
			            	-  IncludeTags: (*bool)(<nil>),
			            	+  IncludeTags: (*bool)(true),
			            	   RequireFullWindow: (*bool)(true),
			Test:       	TestMonitorCreateAndDelete
	monitors_test.go:27: 
			Error Trace:	monitors_test.go:27
			Error:      	Not equal: 
			            	expected: &datadog.Monitor{Creator:(*datadog.Creator)(0xc420246720), Id:(*int)(0xc42024a828), Type:(*string)(0xc420218450), Query:(*string)(0xc420218430), Name:(*string)(0xc420218440), Message:(*string)(0xc420218420), OverallState:(*string)(0xc420218460), OverallStateModified:(*string)(nil), Tags:[]string{}, Options:(*datadog.Options)(0xc420160180), State:datadog.State{Groups:map[string]datadog.GroupData(nil)}}
			            	actual  : &datadog.Monitor{Creator:(*datadog.Creator)(0xc420246ce0), Id:(*int)(0xc420162e80), Type:(*string)(0xc420218aa0), Query:(*string)(0xc420218a30), Name:(*string)(0xc420218a50), Message:(*string)(0xc420218a40), OverallState:(*string)(0xc420218a90), OverallStateModified:(*string)(nil), Tags:[]string{}, Options:(*datadog.Options)(0xc420160600), State:datadog.State{Groups:map[string]datadog.GroupData(nil)}}
			            	
			            	Diff:
			            	--- Expected
			            	+++ Actual
			            	@@ -29,3 +29,3 @@
			            	   ThresholdWindows: (*datadog.ThresholdWindows)(<nil>),
			            	-  IncludeTags: (*bool)(<nil>),
			            	+  IncludeTags: (*bool)(true),
			            	   RequireFullWindow: (*bool)(true),
			Test:       	TestMonitorCreateAndDelete
```

**Comment by dharada1** [comment]
↑ fixed

**Comment by ojongerius** [comment]
Fixed by #210
